src/Umbraco.Web.UI.Client/src/common/services/tinymce.service.js

```diff
@@ -118,15 +118,14 @@ function tinyMceService(dialogService, $log, imageHelper, $http, $timeout, macro
                var data = {
                    alt: img.altText || "",
                    src: (img.url) ? img.url : "nothing.jpg",
-                   rel: img.id,
+                   id: img.id,

```

Comment:
It could potentially cause invalid markup, if the editor has used multiple times on page, e.g. with richtext editors in a grid-editor.
I think that is why `data-id` has been added instead.

But I think it already was fixed in this PR https://github.com/umbraco/Umbraco-CMS/pull/496

Issue: http://issues.umbraco.org/issue/U4-5480